module/zfs/dmu.c

```diff
@@ -2825,3 +2853,7 @@ ZFS_MODULE_PARAM(zfs, zfs_, dmu_offset_next_sync, INT, ZMOD_RW,
 /* CSTYLED */
 ZFS_MODULE_PARAM(zfs, , dmu_prefetch_max, UINT, ZMOD_RW,
 	"Limit one prefetch call to this size");
+
+/* CSTYLED */
+ZFS_MODULE_PARAM(zfs, , dmu_ddt_copies, UINT, ZMOD_RW,
+	"Override copies= for dedup objects");
```

Comment:
You'll want to add an entry for `dmu_ddt_copies` to the `zfs.4` man pages.  Also, since you mentioned it is to be used for testing, you may want to add a warning in the man page like:

`dmu_ddt_copies` _is designed for testing, and not recommended to be used in production_